index.html

```diff
@@ -25,7 +27,7 @@ <h3 class="logo">Henry Cadena</h3>
       </nav>
     </header>
     <main class="container-main">
-      <img class="hero-image" src="assets/background-mobile-version.svg" alt="background">
+      <!-- <img class="hero-image" src="assets/background-mobile-version.svg" alt="background"> -->
```

Comment:
- [ ] [OPTIONAL] Kindly remove the commented-out code from your code source. It would make your code clean and maintainable 🚯

Reply:
Thank you very much @rica213, all the best